python/ycm/vimsupport.py

```diff
@@ -301,6 +301,23 @@ def EscapedFilepath( filepath ):
   return filepath.replace( ' ' , r'\ ' )
 
 
+def TryJumpLocationInOpenedTab( filename, line, column ):
+  bufnr = GetBufferNumberForFilename( filename, False )
+  try:
+    tab, win = next( iter( [ ( tab, win ) for tab in vim.tabpages
+                                          for win in tab.windows
+                                          if tab != vim.current.tabpage
+                                          and win.buffer.number == bufnr ] ) )
+    vim.current.tabpage = tab
+    vim.current.window = win
+    vim.current.window.cursor = ( line, column - 1 )
```

Comment:
Did you double-check that the column calculation here is correct? If you did, add a comment explaining why `column - 1` should be used instead of just `column`.